src/engine/QueryExecutionContext.h

```diff
@@ -23,6 +23,8 @@
 #include <string>
 #include <vector>
 
+#include "../util/websocket/Common.h"
```

Comment:
Regarding import rules. In this header there are quite a lot of imports using the `#include <>` syntax. Is there any difference between these two or is this just inconsistent behaviour?